setup.cfg

```diff
@@ -77,6 +77,7 @@ tests =
   pytest-doctestplus>=0.5 # We require the newest version of doctest plus to use +IGNORE_WARNINGS
   pytest-mock
   pytest-mpl>=0.12 # First version to support our figure tests
+  pytest-intercept-remote>=1.2
```

Comment:
Adding the plugin is enough, no need for a marker or keyword to pytest?

Reply:
Actually keywords like `--intercept-remote` and `--remote-status` are added via the plugin; markers are not required.